data-common.json

```diff
@@ -25,11 +25,13 @@
   },
   "firefox": {
     "developer": {
+      "versions": ["firefox52", "firefox53"],
```

Comment:
I've tried that twice working on this PR but that doesn't work well
https://github.com/kangax/compat-table/pull/989#issuecomment-269059385